packages/namekit-react/src/components/Input.tsx

```diff
@@ -57,6 +52,17 @@ export const Input: React.FC<InputProps> & { Slot: React.FC<SlotProps> } = ({
     (child: any) => child.type === Input.Slot,
   );
 
+  const variantClasses = {
+    primary: cc([
+      "nk-text-black nk-border-gray-300 nk-shadow-[0_1px_2px_0_rgba(0,0,0,0.05)] hover:nk-border-gray-400 has-[input:focus]:nk-border-gray-600 has-[input:focus:hover]:nk-border-gray-600",
+      disabled ? "nk-bg-gray-50" : "nk-bg-white",
+    ]),
+    secondary: [
+      "nk-border-gray-200 hover:nk-border-gray-300 has-[input:focus]:nk-border-gray-400 has-[input:focus:hover]:nk-border-gray-400",
+      disabled ? "nk-bg-gray-200" : "nk-bg-gray-100",
```

Comment:
@eduramme Suggest we make a few styling updates to the disabled variation.

For reference, here's a screenshot of the `secondary` variation:

![CleanShot 2024-09-20 at 23 25 55](https://github.com/user-attachments/assets/d5d3bec2-f58c-4777-8a8c-5cfdb01d293e)

And here's a screenshot of the `disabled` variation:

![CleanShot 2024-09-20 at 23 26 37](https://github.com/user-attachments/assets/016258f4-f9aa-4c44-a945-4de598d4239c)

See the issues here?

1. The background color of the disabled case should be darker than the secondary case.
2. The font color of the disabled case should also be made darker. It shouldn't feel so "alive".
3. Perhaps the border styles should also be adjusted? Appreciate your suggestions.

Also, am I correct to understand that there's distinct disabled styling for both primary and secondary variations? If so, please create distinct stories for "Primary Disabled" and "Secondary Disabled". Thanks :+1: We always want to make it easy to visualize the distinct states of these UI components.

Reply:
@lightwalker-eth Nice! great catch.  Yes, we have two distinct disabled styles: one for primary and one for secondary. I hadn’t created a story for the secondary disabled version yet, but I’ve just added it now :)